javascript/javascript2/week1/hyf-bay-homework/index.html

```diff
@@ -7,7 +7,7 @@
       rel="stylesheet"
     />
   </head>
-  <body>
+  <body id = "body">
```

Comment:
Good idea to add IDs to elements! Keep in mind though that in this particular case, ideally there should always be only one `<body>` element in an HTML document. 🤓